frontend/src/routes/Search.tsx

```diff
@@ -518,25 +547,134 @@ const SearchEvent: React.FC<SearchEventProps> = ({
                         audioOnly,
                     },
                 }}
-                css={{
-                    outline: `1px solid ${COLORS.neutral15}`,
-                    minWidth: 270,
-                    width: 270,
-                    marginLeft: "auto",
-                    [screenWidthAtMost(800)]: {
-                        minWidth: 240,
-                        width: 240,
-                    },
-                    [screenWidthAtMost(BREAKPOINT_MEDIUM)]: {
-                        maxWidth: 400,
-                        margin: "0 auto",
-                    },
-                }}
+                css={thumbnailCss}
             />
         </Item>
     );
 };
 
+const thumbnailCss = {
+    outline: `1px solid ${COLORS.neutral15}`,
+    minWidth: 270,
+    width: 270,
+    marginLeft: "auto",
+    [screenWidthAtMost(800)]: {
+        minWidth: 240,
+        width: 240,
+    },
+    [screenWidthAtMost(BREAKPOINT_MEDIUM)]: {
+        maxWidth: 400,
+        margin: "0 auto",
+    },
+};
+
+
+type ThumbnailInfo = {
+    readonly audioOnly: boolean;
+    readonly isLive: boolean;
+    readonly thumbnail: string | null | undefined;
+}
+type SearchSeriesProps = {
+    id: string;
+    title: string;
+    description: string | null;
+    thumbnailInfo: readonly ThumbnailInfo[] | undefined;
+}
+
+const SearchSeries: React.FC<SearchSeriesProps> = ({ id, title, description, thumbnailInfo }) =>
+    <Item key={id} link={DirectSeriesRoute.url({ seriesId: id })}>
+        <WithIcon Icon={LuLibrary} iconSize={28} hideIconOnMobile>
+            <div css={{
+                color: COLORS.neutral90,
+                marginRight: "clamp(12px, 4vw - 13px, 40px)",
+                display: "flex",
+                flexDirection: "column",
+                minWidth: 0,
+            }}>
+                <h3 css={{
+                    color: COLORS.primary0,
+                    marginBottom: 6,
+                    fontSize: 17,
+                    lineHeight: 1.3,
+                    ...ellipsisOverflowCss(2),
+                }}>{title}</h3>
+                {description && <SmallDescription
+                    text={description}
+                    lines={3}
+                />}
+            </div>
+        </WithIcon>
+        <ThumbnailStack {...{ thumbnailInfo, title }} />
+    </Item>
+;
+
+type ThumbnailStackProps = Pick<SearchSeriesProps, "title" | "thumbnailInfo">
+
+const ThumbnailStack: React.FC<ThumbnailStackProps> = ({ thumbnailInfo, title }) => <div css={{
+    ...thumbnailCss,
+    outline: 0,
+    display: "grid",
+    gridAutoColumns: "1fr",
+    "> div": {
+        outline: `1px solid ${COLORS.neutral10}`,
+        outlineOffset: -.5,
```

Comment:
Why did you add that half pixel offset by the way? That essentially makes the outline half a pixel wide, which is super fine and doesn't look that good, even at 200% scaling on my 4k monitor I think. I would probably just remove the offset?

Also, looking into this, there seem to be some rendering artifacts similar to https://github.com/elan-ev/tobira/issues/765. But maybe distinct from that issue. They do happen as well in our normal thumbnails though. The `object-fit: contain` and `background-color: black` seem to be the problem. Happens in FF and chrome. But yeah, this PR doesn't have to fix this problem.